File: ATIVIDADEI/exercicio6.py
comprimento1 = int(input("Digite o  comprimento do primeiro lado: "))
comprimento2 = int(input("Digite o  comprimento do segundo lado: "))
comprimento3 = int(input("Digite o  comprimento do terceiro lado: "))

if comprimento1 == comprimento2 == comprimento3: # duvida: por que a lógica não funcionou quando verifiquei primeiro o isóceles?
    print("O triângulo é equilátero.")
elif comprimento1 == comprimento2 or comprimento1 == comprimento3 or comprimento2 == comprimento3:
    print("O triângulo é isóceles.")
else:
    print("O triângulo é escaleno.")

File: 2023/ATIVIDADEI/exercicio6.cpp
#include <iostream>
#include <conio.h>

using namespace std;

int main(){
    int comprimento1, comprimento2, comprimento3;

    cout << "Digite o primeiro comprimento" << endl;
    cin >> comprimento1;
    cout << "Digite o segundo comprimento" << endl;
    cin >> comprimento2;
    cout << "Digite o terceiro comprimento" << endl;
    cin >> comprimento3;

    if (comprimento1 == comprimento2 && comprimento1 == comprimento3){
        cout << "O triângulo é equilátero." << endl;
    } if (comprimento1 == comprimento2 || comprimento1 == comprimento3 || comprimento2 == comprimento3) {
        cout << "O triângulo é isóceles." << endl;
    } else {
        cout << "O triângulo é escaleno." << endl;
    };

    cout << "Digite qualquer tecla para sair. ";
    getch();
    return 0;
}